go.mod

```diff
@@ -1,17 +1,17 @@
 module viam-filtered-camera-selective-sync
 
-go 1.23.2
+go 1.23.8
 
 require (
-	go.viam.com/rdk v0.48.2
-	go.viam.com/utils v0.1.111
+	go.viam.com/rdk v0.71.0
```

Comment:
How about we bump to https://github.com/viamrobotics/rdk/releases/tag/v0.72.1 the latest rdk release?